core/src/main/resources/lib/hudson/buildListTable.jelly

```diff
@@ -45,7 +45,8 @@ THE SOFTWARE.
                       insert(generateSVGIcon(e.iconName, "${iconSizeClass}"))));
               tr.insert(new Element('td').
                   insert(new Element('a', {class: 'jenkins-table__link model-link', href: '${rootURL}/' + e.parentUrl}).
-                      update(e.parentFullDisplayName)).
+                   insert(new Element('span').
+                      update(e.parentFullDisplayName))).
```

Comment:
This part improves break lines in the "Build History page".
### Before
![Before](https://user-images.githubusercontent.com/71805759/169387025-8ef21c6d-ba31-46b7-9002-b355bea56522.PNG)
### After
![After](https://user-images.githubusercontent.com/71805759/169387056-9a9384ac-6e95-49d9-a4e4-842997eba1d2.PNG)